docs/minikube-tutorial.md

```diff
@@ -0,0 +1,196 @@
+# Deploy TiDB in the minikube cluster
+
+This document describes how to deploy a TiDB cluster in the [minikube](https://kubernetes.io/docs/setup/minikube/) cluster.
+
+## Table of Contents
+
+- [Start a Kubernetes cluster with minikube](#start-a-kubernetes-cluster-with-minikube)
+  * [What is minikube?](#what-is-minikube)
+  * [Install minikube and start a Kubernetes cluster](#install-minikube-and-start-a-kubernetes-cluster)
+  * [Install kubectl to access the cluster](#install-kubectl-to-access-the-cluster)
+- [Install TiDB operator and run a TiDB cluster with it](#install-tidb-operator-and-run-a-tidb-cluster-with-it)
+  * [Install helm](#install-helm)
+  * [Install TiDB operator in the Kubernetes cluster](#install-tidb-operator-in-the-kubernetes-cluster)
+  * [Launch a TiDB cluster](#launch-a-tidb-cluster)
+  * [Test TiDB cluster](#test-tidb-cluster)
+  * [Delete TiDB cluster](#delete-tidb-cluster)
+- [FAQs](#faqs)
+  * [TiDB cluster in minikube is not responding or responds slow](#tidb-cluster-in-minikube-is-not-responding-or-responds-slow)
+
+## Start a Kubernetes cluster with minikube
+
+### What is minikube?
+
+[Minikube](https://kubernetes.io/docs/setup/minikube/) can start a local
+Kubernetes cluster inside a VM on your laptop. It works on macOS, Linux, and
+Windows.
+
+### Install minikube and start a Kubernetes cluster
+
+See [Installing Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) to install
```

Comment:
Add minimal version requirement of minikube.

Reply:
done